Library/Homebrew/test/test_dependency_collector.rb

```diff
@@ -39,7 +39,8 @@ def test_dependency_tags
 
   def test_requirement_creation
     @d.add :x11
-    assert_instance_of X11Requirement, find_requirement(X11Requirement)
+    assert_instance_of X11Requirement, find_requirement(X11Requirement) if OS.mac?
+    assert_instance_of XorgRequirement, find_requirement(XorgRequirement) unless OS.mac?
```

Comment:
I'd prefer one of either
```ruby
if OS.mac?
    assert_instance_of X11Requirement, find_requirement(X11Requirement)
else
    assert_instance_of XorgRequirement, find_requirement(XorgRequirement)
 end
```
or
```ruby
req = OS.mac? ? X11Requirement : XorgRequirement
assert_instance_of req, find_requirement(req)
```
(untested) I'm not sure the latter is actually legal code. Guess we'll find out.